PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -152,11 +163,54 @@ class ArchivedItemsListViewModel: ItemsListViewModel {
                 },
                 UIAlertAction(title: "Yes", style: .destructive) { [weak self] _ in
                     self?.presentedAlert = nil
+                    self?.delete(item: item)
                 }
             ],
             preferredAction: nil
         )
     }
+
+    private func delete(item: ArchivedItem) {
+        Task {
+            try? await source.delete(item: item)
```

Comment:
Do we need to await for the delete, or just react to it right away? A slow network connection, for example, would cause this to "wait".

Reply:
Discussed offline: we optimistically update the UI and then revert back if the operation fails